src/style.css

```
@@ -0,0 +1,125 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  border: none;
+  font-family: "Roboto", sans-serif;
+}
+
+a {
+  color: #fff;
+}
+
+html,
+body {
+  height: 100%;
```

`width: 100vw;` and `width: 100%;` (in this case) it’s basically the same, but in this case make more sense if you use vw/vh.
Check more info here: https://www.w3schools.com/cssref/css_units.asp

And normally you write the x axis first, and the y after :)

```suggestion
  width: 100vw;
  height: 100vh;
```

---

ill change it but i dont understand the differance

---

You shouldn't just do things because I said so, but because you understand, and it makes sense to you.